pkg/operator/controller/dns/controller.go

```diff
@@ -25,7 +25,7 @@ import (
 	splitdns "github.com/openshift/cluster-ingress-operator/pkg/dns/split"
 	logf "github.com/openshift/cluster-ingress-operator/pkg/log"
 	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
-	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	controller "github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
```

Comment:
This change is unnecessary, and I don't expect gofmt would add it.  Was it added by your editor?

Reply:
Removed.